tensorflow/lite/micro/kernels/conv_test.cc

```diff
@@ -409,8 +409,8 @@ TF_LITE_MICRO_TEST(Kernel1x1QuantizedPerChannel) {
 
 TF_LITE_MICRO_TEST(Kernel1x1QuantizedPerChannelRelu6) {
   // conv params:
-  // padding, stride_<width,height>, dilation_<width, height>, activation
-  TfLiteConvParams conv_params = {kTfLitePaddingValid, 1, 1, kTfLiteActRelu6};
+  // padding, stride_<width,height>, activation, dilation_<width, height>
+  TfLiteConvParams conv_params = {kTfLitePaddingValid, 1, 1, kTfLiteActRelu6, 1, 1};
```

Comment:
This looks like a bugfix too, since I assume the dilation would otherwise default to zero instead of one? Did that cause issues?

Reply:
Yes, in op implementation we analyze dilation ratio in addition to other params to decide whether MLI can be applied or reference code should be used. 
We decided it’s more looks like a bug and to extend test coverage of MLI we changed this line and update comment (otherwise it would be confusing). Are zeroes an acceptable case (maybe it's an implicit part of test)?